### ISSUE TITLE
Update to hardkernel v6.1.90

### ISSUE THREAD

**Issue by kyak** [ffaille/alarm-linux-odroid-xu3#1]
Hi @ffaille 

I'm glad I've found your repo. Thanks for your efforts packaging the v6.1 kernel (otherwise I was thinking about doing it myself).

For now I'm using v4.14, but upgraded to the latest v4.14.336 as opposed to v4.14.180 shipped in Arch Linux ARM repo. I've been maintaining my own v4.14 package roughly following upstream v4.14 release cycle, see https://github.com/kyak/linux-odroid-xu3. This used to be on AUR until recently removed by moderators due to being ARM-specific.

My hardware is Odroid HC1 (no HDMI, headless server), so I'm not very demanding to have the latest and greatest kernel. But I'm still willing to switch over to v6.1, since there is some activity at least in hardkernel's repo. Also because I don't like it how systemd reports this kernel as `Tainted: old-kernel` :)

Thus said, I can participate in testing and refining of v6.1 kernel package with the intention of upstreaming this package to Arch Linux ARM.

For now, I'm building the v6.1.90 (I modified your PKGBUILD locally, hence the issue), which is the latest version available from hardkernel. I will try to use this version for the time being.

Since I'm on Odroid HC1, I've removed the odroid-xu3-platform dependency (it's not relevant for the headless server).

To summarize this issue:

* What are your plans on upstreaming this package to Arch Linux ARM (I know you said it in README, is it still the case)?
* Please upgrade to v6.1.90
* Please consider moving odroid-xu3-platform to optdepends for use cases like headless server

**Comment by kyak** [comment]
It seems that v6.1 will be discontinued in favor of v6.6, see https://github.com/hardkernel/linux/pull/435#issuecomment-1992464208

I've still built v6.1.90 locally (had to disable MALI_MILGARD driver, there were compilation errors with gcc-14 -- there is no way this driver could possibly work), but will focus on v6.6 from now on.

**Comment by polyzen** [comment]
@kyak, did you get 6.1.x or 6.6.x to work on your HC1?

`core/linux-odroid-xu3` has been updated to 6.6.36, but it [seems to not be working for HC1](https://github.com/archlinuxarm/PKGBUILDs/pull/2027#issuecomment-2212529779).

**Comment by kyak** [comment]
> @kyak, did you get 6.1.x or 6.6.x to work on your HC1?
> 
> `core/linux-odroid-xu3` has been updated to 6.6.36, but it [seems to not be working for HC1](https://github.com/archlinuxarm/PKGBUILDs/pull/2027#issuecomment-2212529779).

Unfortunately not.. I've built the latest 6.6, then it refused to boot with the same error as in your linked PR.

I didn't have a chance to take a closer look.
